.github/actions/next-stats-action/src/constants.js

```diff
@@ -3,10 +3,8 @@ const os = require('os')
 
 const benchTitle = 'Page Load Tests'
 const workDir = path.join(os.tmpdir(), 'next-stats')
-const mainRepoName = 'main-repo'
-const diffRepoName = 'diff-repo'
-const mainRepoDir = path.join(workDir, mainRepoName)
-const diffRepoDir = path.join(workDir, diffRepoName)
+const mainRepoDir = path.join(os.tmpdir(), 'main-repo')
+const diffRepoDir = path.join(os.tmpdir(), 'diff-repo')
```

Comment:
Why did this get changed? We shouldn't be creating top-level folders in the tmp directory unless they are unique which these aren't really.

Reply:
Let’s use `fs.mkdtemp()` instead


https://nodejs.org/api/fs.html#fspromisesmkdtempprefix-options

Reply:
It's CI so it should be safe anyway (and it was so far), but it's anti-pattern anyway.

Reply:
The PR stats is meant to be able to run locally as well